sklearn_numba_dpex/common/kernels.py

```diff
@@ -172,12 +171,16 @@ def make_sum_reduction_2d_axis1_kernel(size0, size1, work_group_size, device, dt
     minus_one_idx = np.int64(-1)
     two_as_a_long = np.int64(2)
 
-    is_1d = size1 is None
+    if fused_unary_func is None:
```

Comment:
Can you document `fused_unary_func`, please?

Reply:
It's documented in the public function that follows. For readability it would be better I think to swap the definitions of those, but it would have increased the diff masked the true diff.